rust/hyperlane-base/src/db/rocks/hyperlane_db.rs

```diff
@@ -107,6 +108,41 @@ impl HyperlaneRocksDB {
         }
     }
 
```

Comment:
I think we need to update `impl HyperlaneSequenceAwareIndexerStoreReader<InterchainGasPayment> for HyperlaneRocksDB` to fetch stored sequences from the db https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/dan/indexing-decorator/rust/hyperlane-base/src/db/rocks/hyperlane_db.rs#L367-L377